docs/guides/custom-fees.md

```diff
@@ -0,0 +1 @@
+# Custom Fees Guide
```

Comment:
@nud3l what exactly do we want to add here?

Reply:
Just a small screenshot about the fee selector component and how to use it. I'd imagine this page to be very short with a single screenshot and the steps to select the currency.

Reply:
Done